docs/proposed/http-storage-node-protocol.rst

```diff
@@ -509,28 +509,21 @@ For example::
 
   [1, 5]
 
-``GET /v1/immutable/:storage_index?share=:s0&share=:sN&offset=o1&size=z0&offset=oN&size=zN``
-!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+``GET /v1/immutable/:storage_index/:share_number?offset=:offset&length=:length
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-Read data from the indicated immutable shares.
-If ``share`` query parameters are given, selecte only those shares for reading.
-Otherwise, select all shares present.
-If ``size`` and ``offset`` query parameters are given,
-only the portions thus identified of the selected shares are returned.
-Otherwise, all data is from the selected shares is returned.
+Read a contiguous sequence of bytes from one share in one bucket.
```

Comment:
At this point maybe worth reconsidering `Range` requests, and say "multi-range requests need not be supported"?

Reply:
Hm yea I guess that might work.  I kinda thought HTTP required multipart response type for *all* range requests... but it looks like that's wrong and a single range can just receive raw resource bytes as the response (per https://datatracker.ietf.org/doc/html/rfc7233#section-4.1)

Reply:
Yeah I was looking at MDN which is nicer, albeit less authoritative: https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests